libexec/tgenv-install

```diff
@@ -28,8 +28,6 @@ else
 fi
 
 [ -n "${version}" ] || error_and_die "Version is not specified"
-version="$(tgenv-list-remote | grep -e "${regex}" | head -n 1)"
-[ -n "${version}" ] || error_and_die "No versions matching '${1}' found in remote"
```

Comment:
Hello,

I don't understand why you have removed the possibility to use the tag `latest`. Sometimes, you just want the latest version no matter what.